Azure.Core.Pole.Tests/tsp/simple-test.tsp

```diff
@@ -0,0 +1,5 @@
+model Person {
```

Comment:
this model and the model Dog are almost the same. Let's remove the Person model. We dont' need both